.github/workflows/ci.yml

```diff
@@ -56,6 +56,9 @@ jobs:
       - name: check cloud info
         run: ./bin/terramate experimental cloud info
 
+      - name: check cloud sync
```

Comment:
This won't work as trust is temporary and should not be persists as a treat here

Reply:
As all Terramate tests talk just with the fake server, I just had this idea of validating if it can successfully sync a deployment with the real cloud services, similarly to the ªcheck cloud infoª job. Otherwise, it's easy to introduce a change which passes all the tests but is broken.

It's working now.
Do you have an alternative idea?

Reply:
Should be only tested against fake server fmpov as the cloud will be reconfigured at any point

Reply:
by reconfigured, do you mean redeployed?